src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -1305,6 +1305,16 @@ export default class AircraftModel {
             return [false, 'unable to taxi to runway, we have just landed'];
         }
 
+        if (this.fms.departureRunwayModel === runwayModel) {
+            if (this.flightPhase === FLIGHT_PHASE.WAITING) {
+                return [false, 'we\'re already holding short of the runway'];
+            }
+
+            if (this.flightPhase === FLIGHT_PHASE.TAXI) {
+                return [false, 'we\'re already taxiing to that runway'];
```

Comment:
Let's include the runway number here as well, so if someone doesn't notice it immediately, they can notice the red response and have a bit more information to remember what they did wrong.